### PR TITLE
Added configurable script for filrunout from configuration.h

### PR DESCRIPTION
**Pull request by paclema**

### CODE REVIEW

File: Marlin/Configuration.h [hunk 1]
```diff
@@ -417,8 +417,11 @@ const bool Z_PROBE_ENDSTOP_INVERTING = false; // set to true to invert the logic
                                  // In RAMPS uses servo pin 2. Can be changed in pins file. For other boards pin definition should be made.
                                  // It is assumed that when logic high = filament available
                                  //                    when logic  low = filament ran out
-//const bool FIL_RUNOUT_INVERTING = true;  // Should be uncommented and true or false should assigned
-//#define ENDSTOPPULLUP_FIL_RUNOUT // Uncomment to use internal pullup for filament runout pins if the sensor is defined.
+#ifdef FILAMENT_RUNOUT_SENSOR
+  const bool FIL_RUNOUT_INVERTING = true;  // Should be uncommented and true or false should assigned
+  #define ENDSTOPPULLUP_FIL_RUNOUT // Uncomment to use internal pullup for filament runout pins if the sensor is defined.
```
Comment (thinkyhead) [inline Marlin/Configuration.h:422]: Should `ENDSTOPPULLUP_FIL_RUNOUT` be disabled by default, as before?

Comment (paclema) [inline Marlin/Configuration.h:422]: Of course, take a look at line number 416 is comented, so configurations of filamentrunout will not be applied by default

Comment (thinkyhead) [inline Marlin/Configuration.h:422]: `ENDSTOPPULLUP_FIL_RUNOUT` was still disabled by default, even with `FILAMENT_RUNOUT_SENSOR` set. Now when you set `FILAMENT_RUNOUT_SENSOR` it automatically enables `ENDSTOPPULLUP_FIL_RUNOUT`. What I want to know is, why is it better to have this enabled by default?

Comment (paclema) [inline Marlin/Configuration.h:422]: ah sorry for misunderstanding you. It is the normal config for compile. As it's said at FIL_RUNOUT_INVERTING, you should uncomment it too. That why I put it uncommented, of course I've tried it ;)
photo226236329031215060

Comment (thinkyhead) [inline Marlin/Configuration.h:422]: Seems fine, actually, since `ENDSTOPPULLUPS` is also enabled by default.

**Comment by thinkyhead** [comment]
Ok, looks good to me. If you don't mind adding the same lines to the other Configuration.h files, this will be good to go!

**Comment by paclema** [comment]
@thinkyhead all the configuration files have been updated! ;)

**Pull request merged**